pkg/scheduler/framework/plugins/nodeaffinity/node_affinity.go

```diff
@@ -107,25 +107,14 @@ func (pl *NodeAffinity) isSchedulableAfterNodeChange(logger klog.Logger, pod *v1
 	if err != nil {
 		return framework.Queue, err
 	}
-	if !isMatched {
```

Comment:
Maybe it's also worth commenting this code rather than just delete it?
We can just uncomment this code after removing preCheck

Reply:
cc @sanposhiho what do you think?

Reply:
I don't like commented code generally speaking. We're not sure we'd be able to use commented code as it is when we uncomment them. Plus, we can just revisit the previous implementation when we want -especially this case, we actually committed it into master, we can just find it in history.